metadata.json

```diff
@@ -10,23 +10,23 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.25.0 < 6.0.0"
+      "version_requirement": ">= 4.25.0 < 7.0.0"
     },
     {
       "name": "puppet/extlib",
-      "version_requirement": ">= 3.0.0 < 4.0.0"
+      "version_requirement": ">= 3.0.0 < 5.0.0"
     },
     {
       "name": "puppetlabs/apache",
-      "version_requirement": ">= 1.2.0 < 5.0.0"
+      "version_requirement": ">= 1.2.0 < 6.0.0"
     },
     {
       "name": "puppet/mongodb",
       "version_requirement": ">= 1.1.0 < 3.0.0"
```

Comment:
needs bumping, too

Reply:
Already done in https://github.com/theforeman/puppet-pulp/pull/370.  Let me rebase.